src/Streamly/Coreutils/Touch.hs

```diff
@@ -12,15 +12,17 @@ module Streamly.Coreutils.Touch
     (
       Touch
     , touch
-    , setCreateNew
-    , setDeRef
+    , create
+    , followLinks
```

Comment:
Can we organize the exports as;

```
      touch

     -- * Options
    , Touch
    , create
    , followLinks
```

Also, check how haddocks look.

We need to follow this same conventions in all coreutils commands.